src/test/java/hudson/plugins/git/TestGitRepo.java

```diff
@@ -43,7 +40,7 @@ public TestGitRepo(String name, File tmpDir, TaskListener listener) throws IOExc
 		this.name = name;
 		this.listener = listener;
 		
-		envVars = new EnvVars();
+		EnvVars envVars = new EnvVars();
```

Comment:
@MarkEWaite: your comment seems to be gone after rebase. Yes the change is to reduce the scope of envVars. It was used only in constructor.